sensor/common/managedcentral/managed.go

```diff
@@ -0,0 +1,26 @@
+package managedcentral
+
+import (
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	managedCentral bool
+	managedMutex   sync.RWMutex
+)
+
+// IsCentralManaged returns if Central is managed
+func IsCentralManaged() bool {
```

Comment:
I'm a bit confused. Is this not already known from the environment variable?

Reply:
oh is that only set in Central? I feel like Central can set this when it makes related Sensor bundles, right?

Reply:
So we can, but there are older sensors that could be connected that will not have it. Also existing sensors that are upgraded will not have it